### PR TITLE
Add tiling notebook

### PR DESCRIPTION
**Pull request by samet-akcay**
# Description

- This PR adds tiling notebook to both notebooks and docs directories. Despite the support, it was not documented before.

## Changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Refactor (non-breaking change which refactors the code base)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Checklist

- [ ] My code follows the [pre-commit style and check guidelines](https://github.com/openvinotoolkit/anomalib/blob/main/CONTRIBUTING.md) of this project.
- [ ] I have performed a self-review of my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing tests pass locally with my changes

### CODE REVIEW

File: notebooks/100_datamodules/104_tiling.ipynb [hunk 1]
```diff
@@ -0,0 +1,249 @@
+{
```
Comment (ashwinvaidya17) [inline notebooks/100_datamodules/104_tiling.ipynb:1]: <p>typo input <s>iamge.</s> "visualize t tiled" - the?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/anomalib/pull/712/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

**Review by ashwinvaidya17 — COMMENTED** [review]
Thanks! I only have one comment

File: notebooks/100_datamodules/104_tiling.ipynb (hunk 1 above)
Comment (djdameln) [inline notebooks/100_datamodules/104_tiling.ipynb:1]: <p><s>This notebook demonstrates how tiling can be done in anomalib.</s> This notebook demonstrates how tiling works in anomalib, and how tiling can be enabled when training a model using anomalib.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/anomalib/pull/712/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Comment (djdameln) [inline notebooks/100_datamodules/104_tiling.ipynb:1]: <p><s>how many pixels must be crossed in order to take the second tile</s> the spatial distance in pixels between adjacent tiles</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/anomalib/pull/712/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

**Comment by samet-akcay** [comment]
All of the comments have been addressed.

**Review by ashwinvaidya17 — APPROVED** [review]
Thanks

**Pull request merged**